Dockerfile-dev

```diff
@@ -8,8 +8,9 @@ WORKDIR /opt/certbot/src
 
 COPY . .
 RUN apt-get update && \
-    apt-get install apache2 git python3-dev python3-venv gcc libaugeas0 \
-                    libssl-dev libffi-dev ca-certificates openssl nginx-light -y && \
+    DEBIAN_FRONTEND=noninteractive apt-get install apache2 git python3-dev \
```

Comment:
Since you add parts of the recommended config for apt-get under Docker, you should also add the `--no-install-recommends` to keep the install list as short as possible.